packages/next/src/server/config-shared.ts

```diff
@@ -507,14 +506,6 @@ export interface NextConfig extends Record<string, any> {
    */
   serverRuntimeConfig?: { [key: string]: any }
 
-  /**
-   * Next.js automatically polyfills node-fetch and enables HTTP Keep-Alive by default.
-   * You may want to disable HTTP Keep-Alive for certain `fetch()` calls or globally.
-   *
-   * @see [Disabling HTTP Keep-Alive](https://nextjs.org/docs/api-reference/next.config.js/disabling-http-keep-alive)
-   */
-  httpAgentOptions?: { keepAlive?: boolean }
```

Comment:
Let's keep this option to avoid a breaking change.

Instead, [this feature](https://nextjs.org/docs/api-reference/next.config.js/disabling-http-keep-alive) can be implemented with `{ headers: { connection: 'close' } }`, right?